src/diffusers/models/unet_2d_blocks.py

```diff
@@ -47,7 +50,15 @@ def get_down_block(
     resnet_skip_time_act=False,
     resnet_out_scale_factor=1.0,
     cross_attention_norm=None,
+    attn_head_dim=None,
 ):
+    # If attn head dim is not defined, we default it to the number of heads
+    if attn_head_dim is None:
+        logger.warn(
+            f"It is recommend to provide `attn_head_dim` when calling `get_up_block`. Defaulting `attn_head_dim` to {attn_num_heads}."
+        )
+        attn_head_dim = attn_num_heads
+
     down_block_type = down_block_type[7:] if down_block_type.startswith("UNetRes") else down_block_type
```

Comment:
nit: Is it possible to assign the magic number (7) in a suitable variable?

Reply:
not related to this PR